18F/identity-idp
# LG-6291: Add password field with toggled visibility to IdV app password confirmation

**Why**: For feature parity with the existing proofing experience, the user should have the option to toggle visibility of the password field when at the password confirmation step.

**Testing Instructions:**

You will need to set the step as enabled in your local `config/application.yml`:

```yml
development:
  idv_api_enabled_steps: '["password_confirm","personal_key","personal_key_confirm"]'
```

1. Navigate to http://localhost:3000
2. Sign in
3. Navigate to http://localhost:3000/verify
4. Complete proofing flow up to password confirmation step
5. Enter password
6. Observe password text is hidden by default
7. Toggle "Show password"
8. Observe password text is visible

**Screenshots:**

Toggled Hidden|Toggled Visible
---|---
![Screen Shot 2022-05-12 at 10 18 42 AM](https://user-images.githubusercontent.com/1779930/168097487-0e015590-a823-4fc6-9c44-2581556ec387.png)|![Screen Shot 2022-05-12 at 10 18 38 AM](https://user-images.githubusercontent.com/1779930/168097496-823016d1-e21b-476c-992a-d62da61e3a04.png)
